src/components/AccountDashboard/AccountDashboard.tsx

```diff
@@ -64,7 +83,24 @@ function ContributedList({ address }: { address: string }) {
       </span>
     )
 
-  return <ProjectsList projects={contributedProjects} />
+  return (
```

Comment:
here we're replacing the original ProjectsList view with a custom grid of WalletContributionCard components. These show the actual contribution amount from a wallet to a project, instead of just the Project info with its total volume